dev-client/src/store/persistence.ts

```diff
@@ -38,3 +39,9 @@ export const loadPersistedReduxState = () => {
     );
   }
 };
+
+export const patchPersistedReduxState = (state?: Partial<AppState>) => {
+  if (state) {
+    merge(state, {soilId: {soilChanges: {}}});
+  }
+};
```

Comment:
1) I don't think that's necessary. I'd rather be able to test the load/save functionality in isolation and also test this in isolation. I don't think the issue of clients "remembering" to call this method is substantial since there is only one client (the app at startup).
2) Good idea, I'll make that change.